priv/templates/boot_win.eex

```diff
@@ -493,12 +504,12 @@ if exists "%rel_dir%\vm.args.bak" do (
 @echo Extra Opts          %erl_opts%
 @echo.
 @echo Hooks:
-@for %%i in (%hook_dir%\*.d\*.bat) do (
```

Comment:
Hooks are all in a `hook_name.d` directory under the main hook directory. I think this change will break those hooks.